templates/admin_puppet.conf.erb

```diff
@@ -0,0 +1,8 @@
+### File managed with puppet ###
+[puppet]
+
+# Directory where status files for in progress uploads will be stored
+upload_working_dir = ~/.pulp/puppet-uploads
```

Comment:
I don't have a particular preference since I don't know pulp that well. It depends on how often the user would change this value.